draft-demarco-oauth-status-assertions.md

```diff
@@ -247,7 +248,7 @@ the expiration datetime of the Digital Credential;
 - MUST enable the offline use cases by employing validation using
 a cryptographic signature and the cryptographic public key of the
 Credential Issuer.
-- MUST NOT contain personal information about the User who owns
+- SHOULD NOT contain personal information about the User who owns
 the Digital Credential to which the Status Assertion refers.
```

Comment:
what I am really looking for is a clear resolution to #31 here by aligning the language as best we can with that of the regulation

Reply:
Thanks for the pointer.

Let's unpack the requirements:

> not allow providers of electronic attestations of attributes or any other party, after the issuance of the attestation of attributes, to obtain data that allows transactions or user behaviour to be tracked, linked or correlated, or knowledge of transactions or user behaviour to be otherwise obtained, unless explicitly authorised by the user;

The key part of this is last bit.

If the user (holder) provides additional information to the issuer, the assertion can contain new information, including potential tracking.

In the case the holder discloses a status assertion or potentially even as fresh disclosure of a disclosable status assertion.. the holder controls what information other parties receive.

In status assertion the biggest privacy risk comes from issuer tracking, based on when fresh assertions are requested.

Justin mentioned this at the mic line in the oauth session at IETF 120.

We should provide guidance on the privacy trade offs.

Reply:
I've revised the suggestion to match https://github.com/peppelinux/draft-demarco-oauth-status-assertions/pull/61/files#r1681325358